src/utils/reshapedmsbuild.fs

```diff
@@ -784,7 +792,15 @@ module internal ToolLocationHelper =
 //        instance.GetType().GetPropserty(propName, BindingFlags.Public).SetValue(instance, propValue, null)
 
     type ResolveAssemblyReference () =
-        let assembly = Assembly.Load(new AssemblyName("Microsoft.Build.Tasks.v4.0, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a"))
+        let assembly =
+            Assembly.Load(
+                new AssemblyName(
+#if !NETSTANDARD1_5
```

Comment:
Is NETSTANDARD1_5 defined in both the project.json and FSharp.Compiler.Service.fsproj?